src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
         ["so"] = new("Somali", "Af Soomaali", "so", "som", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
-        ["es"] = new("Spanish", "Español", "es", "spa"),
+        ["sq"] = new("Albanian", "Shqip", "sq", "sqi"),
+        ["sr"] = new("Serbian (Cyrillic)", "Српски", "sr", "srp"),
+        ["sr-Latn"] = new("Serbian (Latin)", "Srpski (latinica)", "sr-Latn", "srp-Latn", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["st"] = new("Sotho", "Sotho", "st", "sot", TranslationServices.Google),
         ["su"] = new("Sundanese", "Basa Sunda", "su", "sun", TranslationServices.Google | TranslationServices.Yandex),
-        ["sw"] = new("Swahili", "Kiswahili", "sw", "swa"),
         ["sv"] = new("Swedish", "Svenska", "sv", "swe"),
-        ["tl"] = new("Tagalog", "Tagalog", "tl", "tgl", TranslationServices.Google | TranslationServices.Yandex),
-        ["tg"] = new("Tajik", "тоҷикӣ", "tg", "tgk", TranslationServices.Google | TranslationServices.Yandex),
+        ["sw"] = new("Swahili", "Kiswahili", "sw", "swa"),
         ["ta"] = new("Tamil", "தமிழ்", "ta", "tam"),
-        ["tt"] = new("Tatar", "Татар", "tt", "tat"),
         ["te"] = new("Telugu", "తెలుగు", "te", "tel"),
+        ["tg"] = new("Tajik", "тоҷикӣ", "tg", "tgk", TranslationServices.Google | TranslationServices.Yandex),
         ["th"] = new("Thai", "ไทย", "th", "tha"),
-        ["tr"] = new("Turkish", "Türkçe", "tr", "tur"),
+        ["ti"] = new("Tigrinya", "ትግር", "ti", "tir", TranslationServices.Bing | TranslationServices.Microsoft),
         ["tk"] = new("Turkmen", "Türkmen Dili", "tk", "tuk", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
+        ["tl"] = new("Tagalog", "Filipino", "tl", "tgl", TranslationServices.Google | TranslationServices.Yandex),
+        ["tlh"] = new("Klingon", "tlhIngan Hol", "tlh", "tlh", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["tlh-Piqd"] = new("Klingon (pIqaD)", "Klingon (pIqaD)", "tlh-Piqd", "tlh-Piqd", TranslationServices.Microsoft), // For some reason Bing stopped supporting this language, ty Microsoft
+        ["to"] = new("Tongan", "Lea Fakatonga", "to", "ton", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["tr"] = new("Turkish", "Türkçe", "tr", "tur"),
+        ["ts"] = new("Tsonga", "Xitsonga", "ts", "tso", TranslationServices.Google),
+        ["tt"] = new("Tatar", "Татар", "tt", "tat"),
+        ["tw"] = new("Twi", "Twi", "tw", "twi", TranslationServices.Google),
+        ["ty"] = new("Tahitian", "Reo Tahiti", "ty", "tah", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["udm"] = new("Udmurt", "Удмурт кыл", "udm", "udm", TranslationServices.Yandex),
+        ["ug"] = new("Uighur", "ئۇيغۇرچە", "ug", "uig", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
```

Comment:
This should be changed back to "Uyghur" as it's the preferred spelling.
```suggestion
        ["ug"] = new("Uyghur", "ئۇيغۇرچە", "ug", "uig", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
```

Reply:
[ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) list the primary language name as Uighur and it has Uyghur as the secondary name.

Reply:
Same as Scottish Gaelic. A solution would be to add "Uyghur" as an alias.